.env

```diff
@@ -1,3 +1,10 @@
 AWS_REGION=
 AWS_ACCOUNT=
-SNS_TOPIC=
\ No newline at end of file
+SNS_TOPIC=
+
+NODE_ENV=dev
+PORT=8080
+
+LOG_LEVEL=info
+LOG_FILE=application.log
```

Comment:
👍 this may need a path too, for production. As you might have seen, it goes to a specific folder on the EC2.

` /var/log/digital-paper-edit-api/application.log`

Reply:
Think this will change with winston-cloudwatch in the next commit :)